src/waveform.js

```diff
@@ -20,21 +20,32 @@ export default class Waveform {
     this.boundMonitorAudioTimeupdate = Waveform.monitorAudioTimeupdateCallback.bind(
       this
     );
+
+    this.boundApplyConfig = Waveform.applyConfig.bind(this);
+
+    try {
+      this.port = chrome.runtime.connect(null, { name: "bandcamplabelview" });
+    } catch (e) {
+      if (e.message.includes("Error in invocation of runtime.connect")) {
```

Comment:
Is this string the same in all browser locales? What happens if the primary language is changed?

Reply:
@emcniece presumably, this is probably taken directly from the Chrome reference and I suspect that would have been mentioned if there was an issue. 

The string is used consistently throughout the plugin so I assume if it changes here, it changes consistently throughout the rest of the plugin and thus works fine?

Reply:
I'd be interested to see what other properties `e` has, like if there's a `e.type` or something more machine-level. String matching on a presentation layer object makes me a bit nervous on platforms that support localization, but this being the browser API might make it a bit more stable. I'm not sure how often (if ever) Chrome changes their error messages.

Reply:
Here's an example of an error with type info: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/URIError

Testing `e instanceof [ErrorType]` might be more portable or reliable 👍

Reply:
it sort of looks like this won't even throw an error...

https://developer.chrome.com/docs/extensions/reference/runtime/

Reply:
@emcniece I think for now I am going to keep this, but I am going to add an issue for me to investigate it further. 

It really looks like this error catching pattern is not useful but... this has been in the code for a while so I assume it's stable at least.

Reply:
@emcniece https://github.com/sabjorn/BandcampEnhancementSuite/issues/85